e2e/app/page/data-page.ts

```diff
@@ -113,9 +113,7 @@ export default class DataPage extends AuthenticatedPage {
   async exportToNotebook(datasetName: string, notebookName: string): Promise<void> {
     const resourceCard = new DataResourceCard(this.page);
     const datasetCard = await resourceCard.findCard(datasetName, CardType.Dataset);
-    await datasetCard.getEllipsis().clickAction(EllipsisMenuAction.exportToNotebook, {waitForNav: false});
-
-
+    await datasetCard.clickEllipsisAction(EllipsisMenuAction.exportToNotebook, {waitForNav: false});
```

Comment:
Removed `getEllipsis()` call because it is not needed anymore. 
Same change is done in other places as well.